modules/oe_translation_poetry/src/Plugin/tmgmt/Translator/PoetryTranslator.php

```diff
@@ -567,6 +618,34 @@ protected function getTranslatedJobsByLanguage(ContentEntityInterface $entity):
     return $result ?? [];
   }
 
+  /**
+   * Get a list of Poetry jobs that are completed for a given entity.
+   *
+   * These are the jobs which have been translated by Poetry in a certain
```

Comment:
I suggest:

It only returns jobs that were sent in the same request as the given job.